### PR TITLE
Recovering Cloud8 using Freezer or SSH backups if upgrade fails

### PR DESCRIPTION
**Pull request by chajain**
Fixing the steps in Unplanned Recovery

### CODE REVIEW

File: xml/operations-maintenance-controller-full_recovery_test.xml [hunk 1]
```diff
@@ -543,7 +543,106 @@ ansible-playbook -i hosts/verb_hosts site.yml --limit doc-cp1-c1-m1-mgmt,doc-cp1
        </listitem>
      </orderedlist>
    </section>
-   <section>
+     <section>
+       <title>Cassandra database restore</title>
+       <para>Cassandra database restore</para>
+       <para>Create a script cassandra-restore-extserver.sh on all
+       controller nodes</para>
```
Comment (rtamalin) [inline xml/operations-maintenance-controller-full_recovery_test.xml:550]: all controller nodes where Cassandra runs, which you can determine by running `ansible FND-CDB --list-hosts` on the deployer, from the scratch area.

File: xml/operations-maintenance-controller-full_recovery_test.xml [hunk 2]
```diff
@@ -652,119 +751,23 @@ Start request sent for job 64715c6ce8ed40e1b346136083923260</screen>
 8.0K    /tmp/mysql_restore/winchester
 4.0K    /tmp/mysql_restore/xtrabackup_galera_info</screen>
        </listitem>
+       <listitem>
+         <para>Repeat steps 2-5 on other two controllers </para>
```
Comment (rtamalin) [inline xml/operations-maintenance-controller-full_recovery_test.xml:755]: other two controllers where the MariaDB/Galera database is running.

File: xml/operations-maintenance-controller-full_recovery_test.xml [hunk 3]
```diff
@@ -652,119 +751,23 @@ Start request sent for job 64715c6ce8ed40e1b346136083923260</screen>
...
+       </listitem>
        <listitem>
          <para>Stop &kw-hos; services on the three controllers (replace the hostnames of the controllers in the command)</para>
 <screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
 &prompt.ardana;ansible-playbook -i hosts/verb_hosts ardana-stop.yml --limit doc-cp1-c1-m1-mgmt,doc-cp1-c1-m2-mgmt,doc-cp1-c1-m3-mgmt,localhost</screen>
        </listitem>
        <listitem>
-         <para>Clean the mysql directory and copy the restored backup</para>
+         <para>Clean the mysql directory and copy the restored backup on all three controllers</para>
```
Comment (rtamalin) [inline xml/operations-maintenance-controller-full_recovery_test.xml:763]: all three controllers where MariaDB/Galera database is running.

File: xml/operations-maintenance-controller-full_recovery_test.xml [hunk 4]
```diff
@@ -205,7 +205,10 @@ var/lib/ardana/service.osrc</screen>
      <title>Execute backup of Cassandra</title>
      <para>Execute backup of Cassandra</para>
      <para>Create cassandra-backup-extserver.sh script on all
-     controller nodes</para>
+     controller nodes where Cassandra runs, which can be determined
+     by running this command on deployer</para>
+<screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
+&prompt.ardana;ansible FND-MDB --list-hosts</screen>
```
Comment (rtamalin) [inline xml/operations-maintenance-controller-full_recovery_test.xml:211]: FND-CDB is where Cassandra runs... Apologies if I typo'd last time...
FND-MDB is where MariaDB/Galera runs...

Comment (rtamalin) [reply xml/operations-maintenance-controller-full_recovery_test.xml:211]: Didn't mean to "approve" last time around.

File: xml/operations-maintenance-controller-full_recovery_test.xml [hunk 5]
```diff
@@ -652,119 +754,24 @@ Start request sent for job 64715c6ce8ed40e1b346136083923260</screen>
 8.0K    /tmp/mysql_restore/winchester
 4.0K    /tmp/mysql_restore/xtrabackup_galera_info</screen>
        </listitem>
+       <listitem>
+         <para>Repeat steps 2-5 on other two controllers where the
```
Comment (rtamalin) [inline xml/operations-maintenance-controller-full_recovery_test.xml:758]: Did you want to do an example of using ansible to fine the FND-MDB nodes here too?

Comment (jeremy-moffitt) [reply xml/operations-maintenance-controller-full_recovery_test.xml:758]: this is missing a "the" .... "Repeat steps 2-5 on other two controllers where the"  should have a "the" between "on" and "other".... "Repeat steps 2-5 on the other two controllers where the"

**Pull request merged**